rio_rgbify/mbtiler.py

```diff
@@ -129,7 +129,7 @@ def _tile_worker(tile):
     reproject(
         rasterio.band(src, 1), out,
         dst_transform=toaffine,
-        dst_crs="init='epsg:3857'",
+        dst_crs='epsg:3857',
```

Comment:
This is the best way to do it for web mercator. If the previous worked, it would have only been by accident, as Rasterio was only expecting `+init=epsg:3857` (no quotes inside the string).